scout/server/links.py

```diff
@@ -412,10 +413,36 @@ def get_variant_links(institute_obj, variant_obj, build=None):
         spidex_human=spidex_human(variant_obj),
         str_source_link=str_source_link(variant_obj),
         snp_links=snp_links(variant_obj),
+        marrvel_link=marrvel_link(variant_obj, build),
     )
     return links
 
 
+def marrvel_link(variant_obj, build):
+    """Create link to Marrvel (http://marrvel.org/) for a variant"""
+
+    url_template = "http://marrvel.org/human/variant/{}:{} {}>{}"
+    chrom = variant_obj["chromosome"]
+    start = variant_obj["position"]
+    ref = variant_obj["reference"]
+    alt = variant_obj["alternative"]
+
+    if build == 38:  # liftover is necessary before returning link
+        client = EnsemblRestApiClient()
```

Comment:
Ooh, do we have a timeout on that, and know anything about their rate limits? This can end up a little bit of traffic, at least when someone opens a list of pages to background. But if the are happy receiving loads of requests I suppose its fine. What bothers me more is that it will slow the page load down with as much as this request takes.

Transcript linkout coordinates would be one option, or delaying the liftover until the user clicks the link, like first pointing to an endpoint on Scout for the final redirect - cf the align viewer that doesn't lift over until one clicks RNA-seq. One could also consider like an async request so that we let the rest of the page finish first, and have like a js that pulls this into the document when it is ready?

Reply:
It's a good point. From what I've seen so far, they don't care about the number of request they get from a DNS, as long as they are not bombed with them. I don't think it's our case. I don't think it would slow down the variant page too much as well (variant links are not generated in the variantS page right? That would be a problem otherwise), but I agree that it's best to do the liftover ONLY if the user clicks on the button. I'll change the code to do as you suggested, use a scout entrypoint to try to do the liftover and redirect the user to marrvel (if liftover works)!